examples/model_selection/plot_successive_halving_heatmap.py

```diff
@@ -6,6 +6,9 @@
 :class:`~sklearn.model_selection.HalvingGridSearchCV` and
 :class:`~sklearn.model_selection.GridSearchCV`.
 
+.. meta::
+   :keywords: versionadded0.24
+
```

Comment:
This isn't a highlight as an individual example, it's just a classic accompanying example for a new estimator.

Also, there's already an entry for SH here, so we don't need to give SH more visibility. Same for the other example below.